packages/@aws-cdk/aws-iam/test/policy-document.test.ts

```diff
@@ -19,22 +19,28 @@ describe('IAM polocy document', () => {
     p.addAwsAccountPrincipal(`my${Token.asString({ account: 'account' })}name`);
     p.addAccountCondition('12221121221');
 
-    expect(stack.resolve(p.toStatementJson())).toEqual({ Action:
```

Comment:
Avoid changing existing tests as a principle (even aesthetic). If I see a change in an existing test, it immediately signals to me that there might be a breaking change here.